cocos/2d/CCNode.cpp

```diff
@@ -1797,6 +1798,12 @@ void Node::setAdditionalTransform(Mat4* additionalTransform)
     _transformUpdated = _transformDirty = _inverseDirty = true;
 }
 
+void Node::setAdditionalTransform(const Mat4& additionalTransform)
+{
+    _useAdditionalTransform = true;
+    _additionalTransform = additionalTransform;
+    _transformUpdated = _transformDirty = _inverseDirty = true;
+}
```

Comment:
I think it is better to re-use existing codes, so it is better to invoke `Node::setAdditionalTransform(Mat4* additionalTransform)` here like

```c++
setAdditionalTransform(&additionalTransform);
```